Applications/PixelPaint/BucketTool.cpp

```diff
@@ -58,10 +58,10 @@ static void flood_fill(Gfx::Bitmap& bitmap, const Gfx::IntPoint& start_position,
     while (!queue.is_empty()) {
         auto position = queue.dequeue();
 
-        if (bitmap.get_pixel<Gfx::BitmapFormat::RGBA32>(position.x(), position.y()) != target_color)
+        if (bitmap.get_pixel(position.x(), position.y()) != target_color)
             continue;
 
-        bitmap.set_pixel<Gfx::BitmapFormat::RGBA32>(position.x(), position.y(), fill_color);
+        bitmap.set_pixel(position.x(), position.y(), fill_color);
```

Comment:
This was specifying the template type to avoid branching on every pixel during a flood fill. The PixelPaint application always uses RGBA32 format, so it's okay to take this shortcut.

Reply:
The PR currently gets rid of the specializations, thus simplifying the interface, and decreasing the chances that it gets misused. If you insist on this microoptimization (does it actually make a difference?), I can add the templates back in, this time depending on `StorageFormat`.

Reply:
Yes, it makes a difference. There might be better ways to implement the flood fill, but with the current algorithm, the extra branching slowed things down noticeably :/